packages/kit/src/runtime/client/renderer.js

```diff
@@ -791,7 +791,7 @@ export class Renderer {
 									state: this.current
 								};
 							}
-
+							if (res.status === 204) return;
```

Comment:
what is this for?

Reply:
When a request endpoint  from shadow page and endpoints return a fallthrough,
then client-side need to  know  the response is a fallthrough. 
We need a flag to tell the client-side that. so I return a 204 status to client-site.